app/views/measures/grouped/_uk.html.erb

```diff
@@ -1,5 +1,7 @@
+<%- uk_import_measures = uk_declarable.import_measures.for_country(@search.country) %>
+
 <!-- UK IMPORT CONTROLS -->
-<% if (collection = uk_declarable.import_measures.import_controls.for_country(@search.country)).present? %>
+<% if (collection = uk_import_measures.import_controls).present? %>
   <%= render partial: 'measures/grouped/table', locals: {
     caption: TradeTariffFrontend::ServiceChooser.uk? ? 'Import controls' : 'UK import controls',
     collection: collection.sort_by(&:key),

```

Comment:
at this point,
the `collection` var seems to be a boolean 🤔 
(from `if (collection = uk_import_measures.import_controls).present?` above)